docs/chart-design-and-style/custom-themes-in-styled-mode.md

```diff
@@ -1,180 +1,64 @@
 Custom themes in styled mode
 ===
 
-Highcharts allows you to create your own custom charts’ theme using native CSS rules or SASS (Syntactically Awesome Style Sheets).
+Highcharts allows you to create your own custom charts’ theme using CSS.
 
-The default CSS file for [styled mode](https://www.highcharts.com/docs/chart-design-and-style/style-by-css) is available as /css/highcharts.css on the root of code.highcharts.com as well as under version folders. This CSS is in turn based on a SCSS file, [highcharts.scss](https://github.com/highcharts/highcharts/blob/master/css/highcharts.scss).
+The default the CSS file for [styled mode](https://www.highcharts.com/docs/chart-design-and-style/style-by-css) is available as [/css/highcharts.css](https://code.highcharts.com/css/highcharts.css) on the root of `code.highcharts.com` as well as under version folders.
 
-Remark
-------
+In the `highcharts.css` file, there are CSS variables for all the colors used in the chart, both for the data and for GUI elements like labels, borders and such. Use them to easily style the chart in the customs theme. As a result, there is no need to overwrite all CSS classes.
 
-Highcharts offers by default three different themes. There are available as CSS files at code.highcharts.com/css/themes :
+In addition to the data colors, Highcharts uses two ranges of colors called the neutral and the highlight colors. These range from a full contrast color with a weight of 100, to low-contrast variations with lower weight. You can use the [Palette Helper](https://www.highcharts.com/samples/highcharts/css/palette-helper) tool to generate these variations.
 
-*   [dark-unica.css](https://code.highcharts.com/css/themes/dark-unica.css)
-    
-*   [sand-signika.css](https://code.highcharts.com/css/themes/sand-signika.css)
-    
-*   [grid-light.css](https://code.highcharts.com/css/themes/grid-light.css)
-    
-
-All these CSS files are generated from SCSS.
 
 How to create your own theme
 ----------------------------
 
-### A. Custom theme using CSS Rules
 
 Follow the steps below to create a custom theme using CSS rules:
 
 1.  Create a CSS file
-    
-2.  Use import to load all default highcharts styles
-    
-3.  Declare all classes which should be modified and adapt them by colors, fonts etc.
-    
-4.  Use the yourtheme.css file on a website.
-    
 
-Example of yourtheme.css
+2.  Use `@import` to load all default highcharts styles
+
+3.  Re-declare CSS variables of the default `highcharts.css` file. For more fine-grained control over each element's appearance, declare individual classes.
+
+
+Example of `yourtheme.css`
+
 
-    
     @import 'https://code.highcharts.com/css/highcharts.css';
-     
-    .highcharts-color-0 {
-      fill: #f45b5b;
-      stroke: #f45b5b;
+
+    /* This will re-declare variables */
+    :root {
+      --highcharts-color-0: #f45b5b;
+    }
+
```

Comment:
How about adding this example too? So we would have:
- all charts change: eg theme
- per chart change: eg different color base for Pie charts (gradient color change?) and a Multi-line chart (different colors per series?)
- change of variables on a class/element (the example below with axis-labels)
- classic CSS change (the example with highcharts-title below)

The example:
```
/* This will overwrite variables for different charts */
#container-1 {
  --highcharts-color-0: #f45b5b;
}

#container-2 {
  --highcharts-color-0: #dedede;
}
```